lightwood/data/timeseries_transform.py

```diff
@@ -109,6 +109,7 @@ def transform_timeseries(
         secondary_type_dict[oby] = dtype_dict[oby]
 
     original_df[f'__mdb_original_{oby}'] = original_df[oby]
+    original_df = _ts_to_obj(original_df, [oby])
```

Comment:
I think the second argument should be `[oby] + tss.historical_columns`, where the latter is a list by default, so adding both should always work.